lib/testsuite/selector-playground/playground-listener.js

```diff
@@ -0,0 +1,60 @@
+const {WebSocketServer} = require('ws');
+const Utils = require('../../utils');
+const {Logger} = Utils;
+
+module.exports = class Websocket {
+  constructor() {
+    this.finishCallback = null;
+    this.portNumber = 8080;
```

Comment:
because the client runs in the chrome extension, right? Then that's not easily possible indeed. We could potentially set it as a config param when we load the extension in Chrome (not sure if that's possible). Either way 8080 is a commonly used port so we should have a different default.

Reply:
Changed the default port to `10096`.